tests/integration/test_charm.py

```diff
@@ -166,6 +166,7 @@ async def test_with_ingress(
     ops_test: OpsTest,
     model: Model,
     flask_app: Application,
+    traefik_app,  # pylint: disable=unused-argument
```

Comment:
Instead of disabling unused arguments, on tests (not on fixtures), `pytest.mark.usefixtures(...)` can be used: https://docs.pytest.org/en/7.1.x/how-to/fixtures.html#use-fixtures-in-classes-and-modules-with-usefixtures